app/Http/Controllers/Admin/UserDisableController.php

```diff
@@ -72,69 +68,4 @@ public function index(Request $request)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.CloseBraceAfterBody</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/UserDisableController.php</br>The closing brace for the class must go on the next line after the body